keps/sig-node/2371-cri-pod-container-stats/README.md

```diff
@@ -432,8 +460,24 @@ message LinuxPodSandboxStats {
 }
 
 // WindowsPodSandboxStats provides the resource usage statistics for a pod sandbox on windows
+//
+// Deprecated: This field is here for backwards compatibility.
+// PodSandboxStats is the preferred field to be used.
 message WindowsPodSandboxStats {
-    // TODO: Add stats relevant to windows.
+}
+
+// SandboxStats provides the resource usage statistics for a Pod Sandbox 
+message SandboxStats {
```

Comment:
this is not a blocker, but a suggestion: what do folks think about compressing this object into the PodSandboxStats above? I find it awkward having a `SandboxStats` object that is singularly nestled in the top level `PodSandboxStats`, especially when all of the other APIs reference sandboxes as `PodSandbox`. @bobbypage @jsturtevant @marosset

Reply:
sorry this just occurred to me, I thought of it when reviewing the corresponding PR

Reply:
I had a similar thought.  It was a bit strange to have `PodSandboxstats.SandboxStats`

Reply:
updated

Reply:
no longer the direction being taken